sympy/printing/pycode.py

```diff
@@ -773,11 +773,14 @@ def _print_Pow(self, expr, rational=False):
         return self._hprint_Pow(expr, rational=rational, sqrt='numpy.sqrt')
 
     def _print_Min(self, expr):
-        return '{0}(({1}))'.format(self._module_format('numpy.amin'), ','.join(self._print(i) for i in expr.args))
+        mini =  '{0}(({1}))'.format(self._module_format('numpy.amin'), ','.join(self._print(i) for i in expr.args))
+        mini = mini[0:len(mini)-1]+ ", axis=0)"
```

Comment:
I think that this is a weird way to post process the strings.
You should better use `'{0}(({1}), axis=0)'.format`

Reply:
I am making that change.